content/blog/2020-07-07-cml-release.md

```diff
@@ -0,0 +1,182 @@
+with **GitHub Actions** and **GitLab CI**. We've created functions to:
+
+1. Generate informative reports on every Pull/Merge Request with metrics, plots,
+   and hyperparameters changes.
+2. Provision GPU\CPU resources from cloud service providers (**AWS, GCP, Azure,
+   Ali**) and deploy CI runners using **docker-machine**.
+3. Bring datasets from cloud storage to runners for model training (using
+   **DVC**), as well as save the resulting model in cloud storage.
+
+![Auto-generated metrics-driven report in GitLab Merge Request](/uploads/images/2020-07-07/cml-report-metrics.png)
+
+The workflow and visual reports are customizable by modifying the CI
+configuration file in your GitHub `./github/workflows/*.yaml` or GitLab
+`.gitlab-ci.yml` project. Use CML functions in conjunction with your own ML
+model training and testing scripts to create your own automated workflow and
+reporting system.
+
+```yaml
+# GitLab workflow in '.gitlab-ci.yml' file
+
+stages:
+  - cml_run
+
+cml:
+  stage: cml_run
+  image: dvcorg/cml-py3:latest
+  script:
+    - dvc pull data --run-cache
+
+    - pip install -r requirements.txt
+    - dvc repro
+
+    # Compare metrics to master
+    - git fetch --prune
+    - dvc metrics diff --show-md master >> report.md
+
+    # Visualize loss function diff
+    - dvc plots diff --target loss.csv --show-vega master > vega.json
+    - vl2png vega.json | cml-publish --md >> report.md
+    - dvc push data --run-cache
+    - cml-send-comment report.md
+```
+
+![Hyperparameter change with a result image in GitHub Pull request report](/uploads/images/2020-07-07/cml-report-params.png)
+
+In this example all the CML functions are defined in the **docker images** that
+is used in the workflow - `dvcorg/cml-py3`. Users can specify any docker image.
+The only restriction is that the CML library need to be installed to enable all
+the CML commands for the reporting and graphs:
+
+```bash
+npm i @dvcorg/cml
+```
+
+Examples of docker images can be found in `docker` directory of the CML the
+repository: [CML repository](https://github.com/iterative/cml).
+
+As you can see, CML is based on the assumption that MLOps can work with
+traditional engineering tools. It shouldn't require an entirely separate
+platform. We're excited about a world where DevOps practitioners can work
+fluently on both software and ML aspects of a project.
+
+## 4. The relationship between CML and DVC
+
+CML and DVC are related projects under the umbrella of the same team, but will
+have separate websites and independent development. The CML project is hosted on
+a new web site: https://cml.dev. The source code and issue tracker is on GitHub:
+https://github.com/iterative/cml
+
+For support and communications, the DVC Discord server is still the place to go:
+https://dvc.org/chat We've made a new `#cml` channel there to discuss CML, CI/CD
+for ML and other MLOps related questions.
+
+## 5. Conclusion
```

Comment:
Completely up to you but are the numbers in each header useful? 🙂

I have no strong opinion at all but I think that's what's causing the anchor link bug (cc @rogermparent) — but that shouldn't prevent the author from using them if he prefers (we should fix the bug anyway).